### PR TITLE
Add diagrams

### PR DESCRIPTION
**Pull request by lucymk**
- Added diagrams to selected stages of workshop to represent process more visually, relates to #39 
- Edited description of how to access repo's live github page

### CODE REVIEW

**Comment by Amirk390** [comment]
https://github.com/foundersandcoders/master-reference/issues/822

#42

File: README.md [hunk 1]
```diff
@@ -187,6 +192,8 @@ You should never merge your own pull requests. A PR gives the rest of your team
 
   <img src="images/pull-request-tab.png" width="500" height="auto" alt="pull request tab button">    
 
+Now your remote repo looks like this: </br>
```
Comment (Amirk390) [inline README.md]: i dont think the `</br>` doing anything :sweat_smile: 
either way i cant see a space between the image and this line

File: README.md [hunk 2]
```diff
@@ -75,13 +75,15 @@ There are many types of workflow. At FAC, we use [the GitHub flow](https://guide
...
 2. Leave the master branch by switching to the new branch you have just created.
 
 ```sh
 $ git checkout create-heading-with-shadow
 ```
-
+</br>
+<img src="images/step3(2)GitFlow.png" width="300" height="auto" alt="repo visual after step 1">
 <!-- ***********************************************************-->
 ## Step 4 - Write enough HTML & CSS to satisfy the requirements
```
Comment (Amirk390) [inline README.md]: For some reason in the actual file the step 4 title isn't working :disappointed: 

Here how it looks from my screen:

image

**Comment by eliascodes** [comment]
@lucymk this is really great and actually more than I had expected to be done, thanks! What did you use to create the diagrams? Is it possible to commit the source files that generated them so we can easily make edits in future?

**Comment by lucymk** [comment]
@eliascodes thanks! I used draw.io, I've just added the link to the source file in the images folder so it can now be opened via draw.io, I'm not sure if there's an easier way to link them or not. They're a bit messy but better than nothing I suppose 🙈

**Pull request merged**